lib/commands/file-movement.js

```diff
@@ -322,23 +332,20 @@ function isDocuments (remotePath) {
  * @param {Object} device - The device object, which represents the device under test.
  *                          This object is expected to have the `udid` property containing the
  *                          valid device ID.
+ * @returns {Array} A list of bundle ids
+ * @throws {Error} If there was an error while calling ifuse command
  */
 async function getAvailableBundleIds (device) {
   await verifyIFusePresence();
-  const list = [];
   try {
-    const apps = await exec('ifuse', ['-u', device.udid, '--list-apps']);
-    for (const app of apps.split(EOL)) {
-      const bundleId = app.split(',')[0].trim();
-      if (!_.isEmpty(bundleId)) {
-        list.push(bundleId);
-      }
-    }
+    const { stdout } = await exec('ifuse', ['-u', device.udid, '--list-apps']);
+    return stdout.split(EOL)
+      .map((app) => app.split(',')[0].trim())
+      .filter((app) => !_.isEmpty(app));
   } catch (e) {
     log.errorAndThrow(`Cannot get a list of bundleIds` +
```

Comment:
Since this is a helper function, which result is only used to build an error message, then I assume it would be better to just print a warning there and return an empty array instead